test/integration_tests/conftest.py

```diff
@@ -33,23 +32,29 @@ def ctc_decoder():
 
 
 _FILES = {
-    'en': 'Lab41-SRI-VOiCES-src-sp0307-ch127535-sg0042.flac',
-    'es': '20130207-0900-PLENARY-7-es_20130207-13_02_05_5.flac',
-    'fr': '20121212-0900-PLENARY-5-fr_20121212-11_37_04_10.flac',
+    'en': (
+        'Lab41-SRI-VOiCES-src-sp0307-ch127535-sg0042.flac',
+        '5005909deb42b30b5f95e0d7109455300de467f8750ff8b3515b1477c31eac24',
```

Comment:
might be nice to cite how these hashes were generated

Reply:
Adding comment that it's SHA256 hash would suffice?

Reply:
sounds good